src/Aspire.Hosting/MongoDB/MongoDBServerResource.cs

```diff
@@ -25,8 +25,8 @@ public class MongoDBServerResource(string name) : ContainerResource(name), IReso
     /// <summary>
     /// Gets the connection string for the MongoDB server.
     /// </summary>
-    public string ConnectionStringExpression =>
-        ConnectionString.ValueExpression;
+    public ReferenceExpression ConnectionStringExpression =>
+        ConnectionString;
 
     /// <summary>
     /// Gets the connection string for the MongoDB server.
```

Comment:
Can we remove these `public ValueTask<string?> GetConnectionStringAsync(CancellationToken cancellationToken)` methods now since they are the same as the DIM on IResourceWithConnectionString?

I guess you remove some in other places. But we should be able to remove all of them.

Reply:
Yep. Missed here.